tests/src/tests/mod.rs

```diff
@@ -351,3 +365,44 @@ async fn test_successful_complicated_flow() {
     assert_eq!(usdc.ft_balance_of(silo2.id()).await, 5_700_000);
     assert_eq!(usdc.ft_balance_of(silo3.id()).await, 7_600_000);
 }
+
+#[tokio::test]
+async fn test_storage_deposit_refund() {
```

Comment:
It might be nice to add some comments about the motivation for this test. Something like "the factory can recover its storage staking after deleting the forwarder account (less the $NEAR spent to perform the delete action)."